src/style.css

```diff
@@ -0,0 +1,123 @@
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("fonts/Avenir.ttc");
+}
```

Comment:
they should have different font-weights, otherwise you wouldn't be able to distinguish them